public/app/config/collections/library/detailFields.js

```diff
@@ -4,6 +4,10 @@
     define(function () {
         return [
             {
+                key: 'type',
```

Comment:
I've added `type` to the detail field specs for a similar reason that I added it to the search views, so that it can be handled consistently, in this case consistency is between the search views and the detail views.